install.sh

```diff
@@ -1,9 +1,68 @@
 #!/bin/bash
 
-# build Docker image
-./build_image.sh
+# --- Includes
+. scripts/utils.sh
 
-# copy anadi.sh in $HOME/bin as anadi
-cp anadi.sh "$HOME/.local/bin/anadi"
+# --- Variables
+CONF_PATH="${HOME}/.config/anadi/"
+BIN_DESTINATION_PATH="$HOME"
+OS=""
 
-echo "Now you can use the command 'anadi'"
+# Makes some checks and sets ENGINE variable
+check_requirements
+
+# OS detection and calling the appropriate install function
+case "$OSTYPE" in
+  darwin*)  
+    OS="OSX" 
+    ;; 
+  linux*)   
+    OS="Linux" 
+    install_linux  # Calling the install function for Linux systems
+    exit 1
+    ;; 
+  msys*|cygwin*|mingw*)  
+    OS="Windows"
+    install_windows  # Calling the install function for Windows systems
+    exit 1
+    ;; 
+  *)        
+    echo "Error: Unknown OS: $OSTYPE"
+    exit 1
+    ;;
+esac
```

Comment:
For Linux and Windows we just copy anadi.sh file but do not build the image.

I suggest to move the function calling outside the switch case after the build_image.

Are you sure build_image function works on Windows  too ?

Reply:
the steps which I imagine in the process :

 - detect the OS
 - check_requirements
     - here we set ENGINE, are you sure that those commands work on windows too?
     
 - build the image based on current OS
     - the bash commands used to create the image work on Windows too?
     
 - build the config path if not exists
     - does Windows support mkdir -p ?
     - have Windows the path "${HOME}/.config/anadi/" ?
     
 - use OS value to call the correct function to install anadi.sh file